modules/lib/lib-node/src/main/resources/lib/xp/node.ts

```diff
@@ -838,7 +827,7 @@ export interface RepoConnection {
 
     findChildren(params: FindChildrenParams): FindNodesByParentResult;
 
-    refresh(mode: RefreshMode): void;
+    refresh(mode?: RefreshMode): void;
```

Comment:
@edloidas why `mode` is optional here? If, yes, then it should be marked as optional in ` NodeHandler` too

Reply:
Because it has always been optional, since it has a default.

Reply:
I agree for JS API, but this is projection on Java, where mode can not be optional.

Reply:
It is optional in interface. In `RepoConnectionImpl` class it has a default value `'ALL'` (we cant have a parameter initializer in interfaces). Keeping `mode` non-optional in the `NodeHandler` makes sure we always pass a value there

Reply:
`RepoConnection.refresh(mode?: RefreshMode)`

Reply:
Yes, `RepoConnection` is the interface that is basically goes to types definitions after compilation. The `mode` here is optional because in actual implementation `RepoConnectionImpl` it has a default value.
`NodeHandler` relates to java code. There we have no default value for the mode, so it must be required here.